core/src/main/java/jenkins/views/JenkinsHeader.java

```diff
@@ -5,14 +5,13 @@
 /**
  * Default {@link Header} provided by Jenkins
  * 
- * @author Ildefonso Montero
  * @see Header
  */
 @Extension(ordinal = Integer.MIN_VALUE)
```

Comment:
Or just
```suggestion
```
since you are already https://github.com/jenkinsci/jenkins/blob/1dd8d32e52489255cc7e91e25b226be138995f82/core/src/main/java/jenkins/views/Header.java#L30

Reply:
with the code as it was this will for the OSS "header is not extended" use case this just return a cached object, whereas now it will create a new `JenkinsHeader` unnecessarily on every call.